opencodelists/settings.py

```diff
@@ -259,3 +260,16 @@
     messages.WARNING: "alert-warning",
     messages.ERROR: "alert-danger",
 }
+
+# EMAIL
+# Anymail
+ANYMAIL = {
+    "MAILGUN_API_KEY": env.str("MAILGUN_API_KEY", default=None),
+    "MAILGUN_API_URL": "https://api.eu.mailgun.net/v3",
+    "MAILGUN_SENDER_DOMAIN": "mg.opencodelists.opensafely.org",
+}
+EMAIL_BACKEND = env.str(
+    "EMAIL_BACKEND", default="django.core.mail.backends.console.EmailBackend"
+)
+DEFAULT_FROM_EMAIL = "you@example.com"
+SERVER_EMAIL = "your-server@example.com"
```

Comment:
We should probably set these to something 😁

Reply:
We should probably also change them on job-server, where I copied the settings from :smile:

Reply:
Are these sensible values?
```
DEFAULT_FROM_EMAIL = "no-reply@opencodelists.org"
SERVER_EMAIL = "tech@opensafely.org"
```

Reply:
I think those make sense 👍